cypress/support/modals.po.ts

```diff
@@ -26,4 +41,93 @@ export class ModalsPo extends BaseComponent {
     directiveEvents: 'demo-modal-events',
     directiveAutoShow: 'demo-modal-auto-shown'
   };
+
+  // temporary placed here
+  isElementVisible(baseSelector: string, elementToFind: string, elemNumber = 0) {
+    cy.get(`${baseSelector} ${elementToFind}`).eq(elemNumber).should('be.visible');
+  }
+
+  isElemTextCorrect(baseSelector: string, itemSel: string, expectedText: string, rowNum = 0) {
+    cy.get(baseSelector).find(itemSel).eq(rowNum).invoke('text')
+      .should('contain', expectedText);
```

Comment:
isItemTextContains(baseSelector: string, itemSelector: string, expectedText: string, rowNum = 0) {
    cy.get(baseSelector).find(itemSelector).eq(rowNum).invoke('text')
      .should('to.contains', expectedText);